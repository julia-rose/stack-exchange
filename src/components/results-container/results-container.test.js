import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import axios from 'axios';
import ResultsContainer from './index';

describe('Results Container', () => {
  const component = shallow(
    <ResultsContainer searchTerm="jquery" />
  );

  it('renders and matches snapshot', () => {
    const component = renderer.create(
      <ResultsContainer searchTerm="jquery" />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});