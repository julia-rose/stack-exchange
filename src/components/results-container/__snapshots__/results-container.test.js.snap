// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Container renders and matches snapshot 1`] = `
<div>
  <h1>
    Results
  </h1>
  <ul />
</div>
`;
